onnx/defs/rnn/defs.cc

```diff
@@ -108,18 +136,19 @@ The order of matrixes `{K, L, D, R, N, C}` is defined as:
     .Input(1, "input",
            "The input sequences packed (and potentially padded) into one 3-D "
            "tensor with the shape of `[seq_length, batch_size, input_size]`.")
-    // TODO: do we want to allow different lengths of sequences in a minibatch?
-    // CuDNN supports it, but not all backend implementations do. One way to
-    // encode would be int-valued tensor denoting lengths of each sequence in
-    // the batch.
     .Input(2, "initial_h",
            "Optional initial value of the hidden. If not specified - assumed "
-           "to be 0. Dimensions `[num_layers * directions, batch_size, "
-           "hidden_size]`")
+           "to be 0. Dimensions `[num_layers * directions, hidden_size]`")
```

Comment:
actually, why did you remove `batch_size`? Cudnn allows to initialize different state per example and it in general makes sense (if you want to continue previous sequence)

Reply:
I don't know it. Reverted.